plugins/catalog/src/components/EntityPageApi/EntityPageApi.tsx

```diff
@@ -28,7 +28,7 @@ export const EntityPageApi: FC<{ entity: Entity }> = ({ entity }) => {
 
   const { value: apiEntities, loading } = useAsync(async () => {
     const a = await Promise.all(
-      ((entity?.spec?.implementedApis as string[]) || []).map(api =>
+      ((entity?.spec?.implementsApis as string[]) || []).map(api =>
```

Comment:
We later changed the name for this, but missed to fix it here 😉 It's already correct in the documentation and entity definition.